### PR TITLE
Readme-general

### PR DESCRIPTION
**Pull request by Nathanael-Rayapin**
The readme for main-web project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,122 @@
...
 
-## Running unit tests
+## Community
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+Be the first to contribute to our project by purchasing and staking our token !
+You'll be able to :
 
-## Running end-to-end tests
+- ✔️ Stake and earn Zion at the same time
```
Comment (etsraphael) [inline README.md]: Wrong name

File: README.md [hunk 2]
```diff
@@ -1,29 +1,122 @@
...
+- ✔️ Use 100% of the platform with full content available to a subscription
+- ✔️ Wiin's certified Artist can be rewarded to personalize monetary gifts
 
-Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+Join the winns comuntiy today and lets grow Zion's worth together 🚀
```
Comment (etsraphael) [inline README.md]: Wrong name again

File: README.md [hunk 3]
```diff
@@ -1,29 +1,122 @@
...
+[![Discord][discord-shield]][discord-url]
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.0.2.
+<br />
+
+<!-- Introduction -->
+
+## Wiins.io
+
+<img src="src/assets/img/readme/readme-introduction.png" alt="Logo" width="" height="">
```
Comment (etsraphael) [inline README.md]: This has to redirect to discord

File: README.md [hunk 4]
```diff
@@ -1,29 +1,122 @@
...
 
-Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+Join the winns comuntiy today and lets grow Zion's worth together 🚀
+<br />
+<br />
+<br />
 
-## Further help
+<hr>
+<img src="src/assets/img/readme/readme-banner.png" alt="Logo" width="" height="">
```
Comment (etsraphael) [inline README.md]: This has to redirrect to discord too

File: README.md (hunk 1 above)
Comment (Nathanael-Rayapin) [reply README.md]: Name changed

File: README.md (hunk 2 above)
Comment (Nathanael-Rayapin) [reply README.md]: name changed

File: README.md (hunk 4 above)
Comment (Nathanael-Rayapin) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Nathanael-Rayapin) [reply README.md]: done

**Pull request merged**